plugins/config_manager.py

```diff
@@ -77,6 +77,8 @@ def configure(self):
                 self.opts.set_disabled or self.opts.add_repo):
             demands.root_user = True
 
+        # sanitize commas https://bugzilla.redhat.com/show_bug.cgi?id=1830530
+        self.opts.crepo = [x.replace(',', '') for x in self.opts.crepo if x is not ',']
```

Comment:
This actually does `repo1,repo2 -> repo1repo2` which doesn't exist. So, it works for `repo1, repo2`, but not for `repo1,repo2`.

Reply:
I changed the way the commas are parsed. Now I hope it should be more clear
Wildcards are also supported as we discussed